app/build.gradle

```diff
@@ -50,7 +50,7 @@ android {
 dependencies {
 
     implementation 'androidx.core:core-ktx:1.7.0'
-    implementation 'androidx.lifecycle:lifecycle-runtime-ktx:2.3.1'
+    implementation 'androidx.lifecycle:lifecycle-runtime-ktx:2.6.1'
```

Comment:
Updated android lifecycle version, this fixed multiple errors on the main activities file due to failed imports.